.github/workflows/dbt_slim_ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           [[ -z "$test" ]] && { echo "Success: All models have a custom schema"; exit 0; } || { echo "Found models without custom schema:"; echo "$test"; exit 1; }
 
       - name: dbt seed
-        run: "dbt seed $PROFILE --select state:modified --exclude tag:prod_exclude tag:remove --state ."
+        run: "dbt seed $PROFILE --select @state:modified --exclude tag:prod_exclude tag:remove --state ."
```

Comment:
remind me the impact of `@` being added?

Reply:
it will run all the upstream seeds of models impacted by the modified changes.

Reply:
Feel free to remove this change if we don't want to push it on `main` ( I think we should, dealing with the fabricated seed changes to trigger them in the CI has become more of a hassle than the extra CI time (<30sec) introduced by this.

Reply:
let's give it a chance & monitor it on main branch